packages/venia-ui/lib/components/Header/switcherItem.js

```diff
@@ -0,0 +1,39 @@
+import React, { useCallback } from 'react';
+import { Check } from 'react-feather';
+import { bool, func, shape, string } from 'prop-types';
+
+import { mergeClasses } from '../../classify';
+import Icon from '../Icon/icon';
+import defaultClasses from './switcherItem.css';
+
+const SwitcherItem = props => {
+    const { active, onClick, switcherItem } = props;
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    const handleClick = useCallback(() => {
+        onClick(switcherItem.code);
+    }, [switcherItem, onClick]);
+
+    const activeIcon = active ? <Icon size={20} src={Check} /> : null;
+
+    return (
+        <button className={classes.root} onClick={active ? null : handleClick}>
```

Comment:
Rather than change the click handler you can just set `disabled={active}`

Reply:
Changed this.

Thank you